bindings/nodejs/lib/types/wallet/account.ts

```diff
@@ -56,8 +56,8 @@ export interface RequiredStorageDeposit {
 export interface NativeTokenBalance {
```

Comment:
Can't comment further above, shouldn't `BaseCoinBalance` fields also be bigint?

Reply:
done

Reply:
Maybe we also should change the voting power, but probably better in another PR for better review.